explorer/ast/statement.h

```diff
@@ -86,11 +86,31 @@ class ExpressionStatement : public Statement {
   Nonnull<Expression*> expression_;
 };
 
+enum class AssignOperator {
+  Plain,
+  Add,
+  Div,
+  Mul,
+  Mod,
+  Sub,
+  And,
+  Or,
+  Xor,
+  ShiftLeft,
+  ShiftRight,
+};
+
+// Returns the spelling of this assignment operator token.
+auto ToString(AssignOperator op) -> std::string_view;
```

Comment:
It seems odd to make such a broadly named function in a header. Does it need to be in the header? Can you call it something more like AssignOperatorToString? Maybe put it as private inside Assign?

Reply:
This is done this way to be consistent with what we do for the parallel `Operator` enum in expression.h.

Reply:
I'd make a similar comment there, #2539. Note it's used from the header.

Reply:
Done.